crates/sui-benchmark/src/options.rs

```diff
@@ -99,6 +99,9 @@ pub struct Opts {
     // Stat collection interval seconds
     #[clap(long, default_value = "10", global = true)]
     pub stat_collection_interval: u64,
+    // Enable stress stat collection
+    #[clap(long, parse(try_from_str), default_value = "false", global = true)]
+    pub stress_stat_collection: bool,
```

Comment:
What is the downside of always enabling this?

Reply:
It [breaks](https://github.com/MystenLabs/sui/actions/runs/3942663537/jobs/6746580409) the sim test, as per marks suggestion it was best to disable it in the tests.